src/Components/MainLayout/Container.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import CardsContainer from '../Cards/CardsContainer'
 import { StyledContainer } from './Styles'
+import AppRouter from '../../Routers/AppRouter'
 
 const Container = () => {
     return (
         <StyledContainer>
-            <CardsContainer/>
+            <AppRouter/>
```

Comment:
This probably wouldn't be used anymore since the Router would be at a higher level. You could instead leave the CardsContainer component in here as it was before